databases/elasticsearch/Portfile

```diff
@@ -1,10 +1,11 @@
 # -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
 
-PortSystem          1.0
+PortSystem                                      1.0
```

Comment:
unnecessary whitespace addition ?

Reply:
Relic of attempt to use github portgroup. Those releases require a full compile, and it’s easier just to download the macOS release directly from elastic.